src/views/ModelPlan/TaskList/ITSolutions/SolutionImplementation/index.tsx

```diff
@@ -506,7 +532,9 @@ const SolutionImplementation = () => {
                               className="margin-right-1"
                               aria-hidden
                             />
-                            {t('dontAdd')}
+                            {isUpdatingStatus
```

Comment:
This submit handler on this button needs to not be called if choosing to not update

```
onClick={() => {
    if (!isUpdatingStatus) handleFormSubmit(values, null, true);
  }}
```

The mutation needs to only be called on not adding, because that actually removed the solution from the need.  When updating, we still want the solution, but don't need to call any mutations

Reply:
I took your suggestion and added another conditional to allow user to go back (behavior that we want) when the click. As your code sits, nothing will happen if user clicks and `!isUpdatingStatus` 
```js
if (isUpdatingStatus) {
  history.goBack();
} else {
  handleFormSubmit(values, null, true);
}
```